share/completions/git.fish

```diff
@@ -1087,6 +1146,36 @@ complete -f -c git -n __fish_git_needs_command -a count-objects -d 'Count unpack
 complete -f -c git -n '__fish_git_using_command count-objects' -s v -l verbose -d 'Be verbose'
 complete -f -c git -n '__fish_git_using_command count-objects' -s H -l human-readable -d 'Print in human readable format'
 
+### daemon
+complete -c git -n __fish_git_needs_command -a daemon -d 'A really simple server for git repositories'
+complete -f -c git -n '__fish_git_using_command daemon' -l strict-paths -d 'Match paths exactly'
+complete -F -c git -n '__fish_git_using_command daemon' -l base-path -d 'Git Root'
+complete -f -c git -n '__fish_git_using_command daemon' -l base-path-relaxed -d 'When looking up with base path fails, try without it'
+complete -F -c git -n '__fish_git_using_command daemon' -l interpolated-path -d 'Construct a path from the given template'
+complete -f -c git -n '__fish_git_using_command daemon' -l export-all -d 'Allow pulling from all directories'
+complete -f -c git -n '__fish_git_using_command daemon' -l inetd -d 'Run as inetd service'
+complete -x -c git -n '__fish_git_using_command daemon' -l listen -d 'Listen on this IP'
+complete -x -c git -n '__fish_git_using_command daemon' -l port -d 'Listen on this port'
+complete -x -c git -n '__fish_git_using_command daemon' -l init-timeout -d 'Connection timeout'
+complete -x -c git -n '__fish_git_using_command daemon' -l timeout -d 'Timeout for each request'
+complete -x -c git -n '__fish_git_using_command daemon' -l max-connections -d 'Maximum parallel clients'
+complete -f -c git -n '__fish_git_using_command daemon' -l syslog -d '--log-destination=syslog'
+complete -x -c git -n '__fish_git_using_command daemon' -l log-destination -a 'stderr syslog none' -d 'Log destination'
+complete -x -c git -n '__fish_git_using_command daemon' -l user-path -d 'Allow ~user notation to be used'
+complete -f -c git -n '__fish_git_using_command daemon' -l verbose -d 'Log all details'
+complete -f -c git -n '__fish_git_using_command daemon' -l reuseaddr -d 'Reuse address when binding to listening server'
+complete -f -c git -n '__fish_git_using_command daemon' -l detach -d 'Detach from shell'
+complete -x -c git -n '__fish_git_using_command daemon' -l reuseaddr -d 'Save the process id in file'
+complete -x -c git -n '__fish_git_using_command daemon' -l user -d 'Change daemon\'s uid'
+complete -x -c git -n '__fish_git_using_command daemon' -l group -d 'Change daemon\'s gid'
+complete -x -c git -n '__fish_git_using_command daemon' -l enable -a 'upload-pack upload-archive receive-pack' -d 'Enable service'
+complete -x -c git -n '__fish_git_using_command daemon' -l disable -a 'upload-pack upload-archive receive-pack' -d 'Disable service'
+complete -x -c git -n '__fish_git_using_command daemon' -l allow-override -a 'upload-pack upload-archive recieve-pack' -d 'Allow overriding site-wide default per repository configuration'
```

Comment:
This still has the `recieve` typo, and since this is an argument, it's important.